### PR TITLE
Dop/post beta 2 updates

### PR DESCRIPTION
**Pull request by doneill**

### CODE REVIEW

**Comment by doneill** [comment]
@eric906 Care to do a cursory review?

File: build.gradle [hunk 1]
```diff
@@ -5,17 +5,19 @@ buildscript {
...
 }
 
 allprojects {
-    repositories {
+
+    final def var = repositories {
         jcenter()
         maven{
-            url 'https://esri.bintray.com/arcgis'
+            url 'http://android:8080/artifactory/arcgis'
```
Comment (eric906) [inline build.gradle]: Isn't this supposed to be https://esri.bintray.com/arcgis ?

File: build.gradle [hunk 2]
```diff
@@ -27,7 +29,7 @@ subprojects{
     afterEvaluate {project ->
         if(project.hasProperty("dependencies")){
             dependencies {
-                compile 'com.esri.arcgisruntime:arcgis-android:100.0.0-beta-2'
+                compile 'com.esri.arcgisruntime:arcgis-android:100.0.0-1334'
```
Comment (eric906) [inline build.gradle]: This would ultimately be "100.0.0-beta-3" right?

File: build.gradle (hunk 1 above)
Comment (doneill) [inline build.gradle]: No, this branch is in active development against our internal repository.

File: build.gradle (hunk 2 above)
Comment (doneill) [inline build.gradle]: Yes, I have updated the README to reflect the public url and public release version this active branch will be working from when released.

Comment (eric906) [inline build.gradle]: Ah. Right. Sorry, I forgot to look at what this file name was. Makes sense.

**Comment by eric906** [comment]
@doneill Review complete. Looks good

**Comment by doneill** [comment]
@eric906 Thanks!

**Pull request merged**